set.mm

```diff
@@ -465888,37 +465904,45 @@ its graph has a given second element (that is, function value).
     $( Describe an implicit one-to-one onto function of two variables.
        (Contributed by Thierry Arnoux, 17-Aug-2017.) $)
     f1od2 $p |- ( ph -> F : ( A X. B ) -1-1-onto-> D ) $=
-      ( wa wsbc va cxp wfn ccnv wf1o wcel wral ralrimivva fnmpt2 syl cv opelxpi
```

Comment:
Comment suggestion: Sufficient condition for a binary function expressed in maps-to notation to be bijective.

Reply:
> I guess the proof is now much longer because the DV conditions in cnvoprab force you to use dummy variables and "rebuild" formulas like x = A -> ( phi[x] <-> phi[A]) (or x = A -> ( B[x] = B[A]) ) ?

Yes, that's correct!

> Did you `MM-PA>minimize_with *` ?

No, I'll try that.